test/views/html/MemberDetailsViewSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Is it documented somewhere to have this file removed prior to final merge & release?